deploy-manage/monitor/autoops/cc-cloud-connect-autoops-faq.md

```diff
@@ -11,7 +11,7 @@ navigation_title: FAQ
 
 Find answers to your questions about AutoOps for self-managed clusters.
```

Comment:
oops commented in wrong place

pls clarify the meaning of self-managed in this page, perhaps in this opening section.